incubator/tensorflow-notebook/values.yaml

```diff
@@ -0,0 +1,25 @@
+# Default values for ack-tensorflow-dev.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+jupyter:
+  image:
+    repository: tensorflow/tensorflow
+    tag: 1.6.0-devel
```

Comment:
Is this a stable release?

Reply:
Yes, devel means it also has development tools.

Reply:
Any reason why you moved away from the official image? I'd prefer you stick with that.

Reply:
I just want to show that we need to keep the two images separately. Ok, I will revert it. Any other changes are required except this?